src/pointfree/init.js

```diff
@@ -0,0 +1,25 @@
+/** @license ISC License (c) copyright 2016 original and current authors */
+/** @author Ian Hofmann-Hicks (evil) */
```

Comment:
why not give yourself credit for creating the file

Reply:
Good catch @bennypowers 😄 @RichardForrester as a standard we accredit the original author for each file at the top